testing/regress/hpcc/regression/regress.py

```diff
@@ -343,20 +343,21 @@ def StartTimeoutThread(self):
         self.timeoutThread.start()
 
     def CheckTimeout(self, cnt,  threadId,  query):
+        defSleepTime=0.1
```

Comment:
That change was intentional, but I can move it into a different JIRA/PR if necessary.